source/extensions/filters/http/jwt_authn/filter.cc

```diff
@@ -29,11 +30,29 @@ void Filter::onDestroy() {
   }
 }
 
+bool isCorsPreflightRequest(const Http::HeaderMap& headers) {
+  return headers.Method() &&
+         headers.Method()->value().getStringView() == Http::Headers::get().MethodValues.Options &&
+         headers.Origin() && !headers.Origin()->value().empty() &&
+         headers.AccessControlRequestMethod() &&
+         !headers.AccessControlRequestMethod()->value().empty();
+}
+
 Http::FilterHeadersStatus Filter::decodeHeaders(Http::HeaderMap& headers, bool) {
   ENVOY_LOG(debug, "Called Filter : {}", __func__);
 
   state_ = Calling;
   stopped_ = false;
+
+  if (isCorsPreflightRequest(headers)) {
```

Comment:
This can be easily done by adding a Requiretment Rule.   I prefer not to hard code this.